app/styles/app/layouts/settings.scss

```diff
@@ -498,3 +498,12 @@ $env-var-value-background: lighten($cement-grey, 20);
     margin-left: 20px;
   }
 }
+
+.ember-tooltip {
```

Comment:
this will probably break some tooltips -  don't add

Reply:
Removed, but it's probably responsible for a tool tip added in enterpise .. but .. removed. Done.